packages/swarm-components/src/TextInput.test.jsx

```diff
@@ -15,6 +15,7 @@ describe('TextInput', () => {
         ['onChange', <TextInput key="7" id="onChange" name="onChange" value="" onChange={mockOnChange} />],
         ['Icon', <TextInput key="8" id="icon" name="icon" value="" iconShape="location-pin" />],
         ['Icon (custom size)', <TextInput key="9" id="iconsize" name="iconsize" value="" iconShape="location-pin" iconSize="m" />],
+		['maxLength (char count)', <TextInput key="10" id="maxLength" name="maxLength" value="foo fooo" maxLength="20"/>],
```

Comment:
looks like the rest of the file needs a Prettier format - space indentation has crept in

Reply:
fixed. ran prettier manually on this file.

Reply:
Hmm, actually running prettier manually (`yarn prettier --write packages/swarm-components/src/shared/CharCount.test.jsx `) did not solve this specific indentation issue, though it did update other things. Looking into it.